manifests/interface.pp

```diff
@@ -112,6 +130,7 @@
   Optional[Integer[1200, 9000]] $mtu = undef,
   Optional[String[1]] $public_key = undef,
   Array[Hash[String[1], Variant[String[1], Boolean]]] $routes = [],
+  Hash[String, Hash[Any, String]] $sections = {},
```

Comment:
According to your example, this seems to be:

```suggestion
  Hash[String, Hash[String, Any]] $sections = {},
```

IIRC, there is also a `Data` type that will only accept actual data and may be more suitable that `Any`.

Reply:
Also, some modules use a parameter name like `$custom_foo` or `$extra_foo` to add custom config.  Maybe somethig to think about before introducing this parameter.

Reply:
Thanks for your feedback. The pipeline is now successful.